### ISSUE TITLE
Use production build for walt-explorer

### ISSUE THREAD

**Issue by JobLeonard** [ballercat/walt#128]
# Feature Request
Build the explorer website with production settings before putting it on-line

## Overview
Currently the walt-explorer is using the dev build on the website. As a result, it's a 5mb JS file. I'm pretty sure it can be reduced to a fraction of that (not to mention run significantly faster) by [instructing `webpack` to create a production build](https://webpack.js.org/guides/production/) before publishing it on the website.

Perhaps the the minification pluginsbreak the walt code, but you should be able to run webpack without them and even then it will improve the payload.

# Impact
Small to medium. Mostly it's just ironic to have a bloated website for a tool to make low-level fast WASM :P

**Comment by ballercat** [comment]
Yes, it's an older version of Webpack (4+ was on its way to release at the time).

There isn't any good reason for 3.x so feel free to upgrade if you are up for it. Thanks!

**Comment by randy7d0** [comment]
Thanks for confirming it! I will give it a try to upgrade it to 4.x

**Comment by randy7d0** [comment]
Hi moderator, can you please give me '**Write**' access in order for me to submit PR for this story. Thanks!

**Comment by ballercat** [comment]
You do not need write access to make a PR. If you cloned the repo locally, then you'll need to create a fork, push your changes to that remote instead and create a PR from there.

https://help.github.com/articles/creating-a-pull-request-from-a-fork/

**Comment by randy7d0** [comment]
I have created the following pull request, please help to review it. Thanks!
https://github.com/ballercat/walt/pull/170

**Comment by randy7d0** [comment]
I could not find anyone whom I can assign as 'Reviewer', however. So, what is the process of doing actual submission (merging to master branch) for this? Please suggest. Thanks!

**Comment by randy7d0** [comment]
I have updated based on feedback. Please have a look again. Thanks!
https://github.com/ballercat/walt/pull/170#pullrequestreview-169071224

**Comment by JobLeonard** [comment]
I think #176 is relevant to this topic ;)
